plugins-extra/query_enhancements/public/query_assist/utils/create_extension.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { HttpSetup } from 'opensearch-dashboards/public';
+import { QueryAssistBar } from '../components';
+import { SearchBarExtensionConfig } from '../../../../../src/plugins/data/public/ui/search_bar_extensions';
+
+export const createQueryAssistExtension = (http: HttpSetup): SearchBarExtensionConfig => {
+  return {
+    id: 'query-assist-ppl',
+    order: 1000,
+    isEnabled: (() => {
+      let agentConfigured: boolean;
```

Comment:
do we want to pull from advanced settings to be able to catch all disable extensions?

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/data/server/ui_settings.ts